packages/store-ui/src/molecules/Modal/Modal.test.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react'
+import { fireEvent, render } from '@testing-library/react'
+
+import Modal from './Modal'
+import Button from '../../atoms/Button'
+import Input from '../../atoms/Input'
+
+const modalTestId = 'store-modal'
+
+const TestModal = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const handleOpen = () => {
+    setIsOpen(true)
+  }
+
+  const onDismiss = () => {
+    setIsOpen(false)
+  }
+
+  return (
+    <>
+      <Button testId="trigger" onClick={handleOpen}>
+        OpenModal
+      </Button>
+      <Modal isOpen={isOpen} testId={modalTestId} onDismiss={onDismiss}>
+        <Input testId="first-input" />
+        <Button testId="first-button" />
+      </Modal>
+    </>
+  )
+}
+
+describe('Modal', () => {
+  it('Test data-store-modal', () => {
+    const { getByTestId } = render(
+      <Modal aria-label="test modal" testId="store-modal" isOpen>
+        Foo
+      </Modal>
+    )
+
+    expect(getByTestId('store-modal')).toHaveAttribute('data-store-modal')
+  })
+
+  it('Test isOpen', () => {
```

Comment:
What if we divide this into 2 tests? 'modal shouldn't be rendered if isOpen is false' and 'modal should be rendered if isOpen is true'

Reply:
Can you explain why you think that should be like this? I didn't get the motivation

Reply:
Yeah, sure! Separating into 2 tests would make us know exactly what is happening if the test fails, for example: if "modal should be rendered if isOpen is true" fails we know that the problem is that the modal isn't opening.
But if you believe this granularity isn't necessary, I would suggest changing the test title to:
```suggestion
  it('Modal should only be rendered if isOpen is true', () => {
```